src/core/energy_storage/electric_storage.jl

```diff
@@ -329,7 +332,14 @@ struct ElectricStorage <: AbstractElectricStorage
             degr = Degradation()
         end
     
+        charge_efficiency = s.rectifier_efficiency_fraction * s.internal_efficiency_fraction^0.5
+        discharge_efficiency = s.inverter_efficiency_fraction * s.internal_efficiency_fraction^0.5
+        dc_charge_efficiency = s.internal_efficiency_fraction^0.5
+        dc_discharge_efficiency = s.internal_efficiency_fraction^0.5
```

Comment:
Should `dc_discharge_efficiency` ever be used if we don't have dc loads? Wouldn't all discharge need to run through the inverter? It looks like this is only used in the renewable_energy_constraints currently

Reply:
True, don't need it. I'll delete.